staging/src/k8s.io/api/scheduling/v1alpha1/types.go

```diff
@@ -24,6 +24,7 @@ import (
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// DEPRECATED - This group version of PriorityClass is deprecated by scheduling/v1/PriorityClass.
```

Comment:
These are external facing so should use `scheduling.k8s.io/v1`

Reply:
Thanks, @liggitt. Done. PTAL.